visualization/bin/svg-example.cpp

```diff
@@ -1,14 +1,13 @@
-#include <visualization/rectangle.hpp>
-#include <visualization/line.hpp>
+#include <fstream>
 #include <visualization/circle.hpp>
+#include <visualization/line.hpp>
+#include <visualization/rectangle.hpp>
 #include <visualization/renderer.hpp>
 
-#include <fstream>
-
 using namespace visualization;
 
-int main() {
-    //Temporary
+int
+    main() {
```

Comment:
Wow! Looks like a bug.. maybe removing the comment would fix it (it's not really useful anyways)

Reply:
So, it's actually a bug. I don't understand why clang-format do this...

Reply:
wait before review or merge after the last commit. I try to find why we have the bug.

Reply:
The bug was `AlwaysBreakAfterReturnType: None` instead `AlwaysBreakAfterReturnType: TopLevelDefinitions`